src/NormalizeSqlValueCapableTrait.php

```diff
@@ -20,15 +20,17 @@
      * @since [*next-version*]
      *
      * @param string|int|float|bool|Stringable $value  The input value.
-     * @param string|Stringable|null           $column Optional column name, used for normalizing for a specific
-     *                                                 column's type.
      *
      * @throws OutOfRangeException If the value cannot be normalized.
      *
      * @return string The normalized value.
      */
-    protected function _normalizeSqlValue($value, $column = null)
+    protected function _normalizeSqlValue($value)
```

Comment:
Why has the `$column` parameter been removed? Because the type will be determined solely based on the type of the value?

Reply:
The parameter was unused in the method, and only one other trait was apparently using that parameter. Since the description of the trait specifically says that the column does not play a part in the normalization process, I assume this parameter was accidentally left in place when in fact it should have been removed, so I took this opportunity to remove it, and tests passed.

Reply:
I understand that this didn't break anything according to the expected behaviour. But I feel that it is still a good idea to supply this argument to give the normalization process more choice. For example, the value may be a string that contains a JSON-encoded object. In this case, it may be required to explicitly normalize the value as JSON, to be recorded in a [`JSON`][mysql-type-json] column type. Or perhaps a numeric string must be normalized into a number.

[mysql-type-json]: https://dev.mysql.com/doc/refman/8.0/en/json.html

Reply:
Yeah, all right. I'll add it back.